examples/plot_classification_example.py

```diff
@@ -0,0 +1,258 @@
+                    0,
+                    alternative="greater",
+                ).pvalue
+                df_pval_list.append(
+                    {
+                        "feature": feature,
+                        "vim": vim,
+                        "model": model,
+                        "pval": pval,
+                        "y_coord": df[(df["vim"] == vim)]["importance"].max(),
+                    }
+                )
+    df_pval = pd.DataFrame(df_pval_list)
+    df_pval = df_pval.query(f"pval < {threshold}")
+    return df_pval
+
+
+threshold = 0.05
+df_pval = compute_pval(df, threshold=threshold)
+
+
+############################################################################
+# Visualization of the results
+# --------------------------------------------------------------------------
+def plot_results(df_importance, df_pval):
+    fig, axes = plt.subplots(1, 2, figsize=(6, 3), sharey=True)
+    for method, ax in zip(["CPI", "PermutationImportance"], axes):
+        df_method = df_importance[df_importance["vim"] == method]
+        legend = ax == axes[-1]
+        sns.stripplot(
+            data=df_pval[df_pval["vim"] == method],
+            x="y_coord",
+            y="feature",
+            hue="model",
+            ax=ax,
+            edgecolor="k",
+            linewidth=1,
+            marker="*",
+            s=10,
+            legend=False,
+            dodge=0.5,
+            orient="h",
+        )
+        sns.boxplot(
+            data=df_method,
+            x="importance",
+            y="feature",
+            hue="model",
+            ax=ax,
+            legend=legend,
+            orient="h",
+        )
+        ax.set_title(method, fontweight="bold")
+        ax.axvline(0, color="k", linestyle="--")
+        ax.set_xlabel("Importance")
+        ax.set_ylabel("")
+        for i in range(len(df_method["feature"].unique())):
+            if i % 2 == 0:
+                ax.axhspan(i - 0.5, i + 0.5, color="gray", alpha=0.33)
+
+    ax = axes[-1]
+    handles, labels = ax.get_legend_handles_labels()
+    handles.append(
+        plt.Line2D(
+            [0],
+            [0],
+            marker="*",
+            color="w",
+            markerfacecolor="w",
+            markeredgecolor="k",
+            markersize=12,
+            label=f"pval < {threshold}",
+            markeredgewidth=1.5,
+        )
+    )
+    ax.legend(
+        handles=handles,
+        labels=labels + [f"pval < {threshold}"],
+        loc="center left",
+        bbox_to_anchor=(1, 0.5),
+    )
+    plt.tight_layout()
+    plt.show()
+
+
+plot_results(df, df_pval)
+
+
+####################################################################################
+# The boxplot shows the importance of each feature, with colors indicating the
+# classifier used. A star marks the features that have a p-value (computed with a
+# t-test) below 0.05. As expected, the spurious feature is not selected by CPI,
+# but is selected by Permutation Importance.
```

Comment:
Can you add a comment between LogReg and SVC model?

Reply:
done